src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsApiService.kt

```diff
@@ -15,4 +17,12 @@ class AdjudicationsApiService(private val adjudicationsApiWebClient: WebClient)
       .retrieve()
       .awaitBody()
   }
+
+  suspend fun getAdjudicationsByBookingId(bookingId: Long, prisonIds: List<String>): List<ReportedAdjudicationDto> {
+    // TODO - switch to simplified non-paged version when available
+    return adjudicationsApiWebClient.get()
+      .uri("/reported-adjudications/booking/{bookingId}?agency={agency}&page={page}&size={size}", bookingId, prisonIds.joinToString(), 0, 1000)
+      .retrieve()
+      .awaitBody<PageReportedAdjudicationDto>().content!!
+  }
 }
```

Comment:
DPS team is writing a simplified version of this, but basic content returned will be similar